core/src/main/java/org/keycloak/representations/account/ClientRepresentation.java

```diff
@@ -6,6 +6,11 @@
 public class ClientRepresentation {
     private String clientId;
     private String clientName;
+    private String description;
+    private boolean internal;
+    private boolean inUse;
+    private String url;
+    private ConsentRepresentation consent;
```

Comment:
@douglaspalmer I believe we should try to simplify this to something like:

```
private List<String> scopes;
private Date createdDate;
```

Even though is technically possible to get these values on the UI side there are 2 reasons for this:
1. Avoid exposing more than we need
2. Avoid including too much logic into the UI if possible. For example, to retrieve the consent names we need a chain of Promises inside the UI, something that the endpoint could easily return.

Reply:
@douglaspalmer this is just a prototype of what I had in mind https://github.com/keycloak/keycloak/commit/cc6ac1367b80a1521b55e323c46f0846adae31cb